### PR TITLE
i18n(es): Multiple pages updates 

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
 Updated different files based on these commits: 
   - [x] `endpoints.mdx` → 8077640390594eab66c6497d5e9146c2e5bef371, 829b66397ad31b2e3921af7f96d5aed7677e8ddf & 33b3229039014f1f9b7ab843300a0c630b1785f3
   - [x] `routing.mdx` → 829b66397ad31b2e3921af7f96d5aed7677e8ddf
   - [x] `getting-started.mdx` → 02d1f39fb615f78322e0a57cb6d5070367d591bf
   - [x] `client-side-scripts.mdx` → 48a942219d7d4d2de3bf6acf9cfc0a997639800b
   - [x] `wordpress.mdx` → 593f66b42d64132bf65f9443ff29042ce34599d6

### CODE REVIEW

**Review by Waxer59 — APPROVED** [review]
LGTM ✔️

File: src/content/docs/es/core-concepts/endpoints.mdx
Comment (kevinzunigacuellar) [inline src/content/docs/es/core-concepts/endpoints.mdx:175]:
```suggestion
<RecipeLinks slugs={["/es/recipes/captcha", "/es/recipes/build-forms-api" ]}/>
```

File: src/content/docs/es/core-concepts/endpoints.mdx
Comment (kevinzunigacuellar) [inline src/content/docs/es/core-concepts/endpoints.mdx:175]:
```suggestion
<RecipeLinks slugs={["es/recipes/captcha/", "es/recipes/build-forms-api/" ]}/>
```

File: src/content/docs/es/core-concepts/endpoints.mdx
Comment (kevinzunigacuellar) [inline src/content/docs/es/core-concepts/endpoints.mdx:175]:
```suggestion
<RecipeLinks slugs={["es/recipes/captcha", "es/recipes/build-forms-api" ]}/>
```

**Comment by kevinzunigacuellar** [comment]
https://github.com/withastro/docs/blob/809419f6efd653b5d394ffb8c820b5588cd4a6be/src/components/RecipeLinks.astro#L16-L19

It seems like if the recipe that is referenced doesn't have a translation it defaults to English which triggers an error in CI. However, the recipes are not yet ready for translation. Is there something we can do in the meantime @delucis ?

**Comment by delucis** [comment]
Just opened #3305 to fix how `<RecipeLinks>` behaves, which should unblock this PR once it’s merged!

**Comment by delucis** [comment]
Fixed! 🥳

**Comment by kevinzunigacuellar** [comment]
thanks a lot @dreyfus92, @Waxer59 and @delucis 💜

**Pull request merged**
